go.mod

```diff
@@ -37,7 +37,7 @@ require (
 	github.com/oapi-codegen/runtime v1.1.1
 	github.com/operator-framework/api v0.27.0
 	github.com/operator-framework/operator-lifecycle-manager v0.27.0
-	github.com/percona/everest-operator v0.6.0-dev1.0.20250131090446-40b6d1d65b10
+	github.com/percona/everest-operator v0.6.0-dev1.0.20250205133424-71ef43450515
```

Comment:
comment to remember to update the dependencies to point out to the feature branch once the related PR is merged